tests/unit/conftest.py

```
@@ -0,0 +1,61 @@
+
+def pytest_runtest_teardown(item):
+	if 'test_config' in str(item.fspath):
+		os.environ['SECATOR_DIRS_REPORTS'] = _REPORTS
+	# Heal any clear_modules() the test performed: purge the (possibly second-generation) secator
+	# modules and restore the collection-time objects, so the next test's runners, drivers and
+	# OutputType classes are all the one generation the test files hold references to. Only do this
+	# when the generation actually changed (sentinel identity) — otherwise we'd also evict modules a
+	# well-behaved test merely imported (e.g. a dynamically-discovered external driver), leaving
+	# discover_external_drivers' @cache pointing at a now-missing module.
+	if _SECATOR_MODULES is not None and \
+			sys.modules.get('secator.output_types') is not _SECATOR_MODULES.get('secator.output_types'):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)